src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
private static final int COLOUR_COUNT = Colors.values().length;
private static final Random random = new Random();

Please move the constants outside of the method. COLOUR_COUNT must be constant since it is a magic number, no need to call enum length multiple times.